index.html

```diff
@@ -10,6 +10,7 @@
   <body>
     <div class="book-container">
       <img class="book-cover" src="assets/book-cover.png" />
+      <img class="badge" src="assets/badge.png" />
```

Comment:
For the use of decorators you could use a pseudo element this because It is recommended to use the least amount of html as possible.